benchmarks/trees/score.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2019, Microsoft CORPORATION. All rights reserved.
```

Comment:
(i think this is is only here from not rebase, but same question)

Reply:
Same answer :)